src/fswAlgorithms/effectorInterfaces/hingedRigidBodyPIDMotor/_UnitTest/test_hingedRigidBodyPIDMotor.py

```diff
@@ -0,0 +1,183 @@
+@pytest.mark.parametrize("P", [0, 5])
+@pytest.mark.parametrize("I", [0, 1])
+@pytest.mark.parametrize("accuracy", [1e-8])
+
+
+# update "module" in this function name to reflect the module name
+def test_hingedRigidBodyPIDMotor(show_plots, thetaR, thetaDotR, theta, thetaDot, K, P, I, accuracy):
+    r"""
+    **Validation Test Description**
+
+    This unit test verifies the correctness of the output motor torque :ref:`hingedRigidBodyPIDMotor`.
+    The inputs provided are reference angle and angle rate, current angle and angle rate, proportional,
+    derivative, and integral gain.
+
+    **Test Parameters**
+
+    Args:
+        thetaR (double): reference angle;
+        thetaDotR (double): reference angle rate;
+        theta (double): current angle;
+        thetaDot (double): current angle rate;
+        K (double): proportional gain;
+        P (double): derivative gain;
+        I (double): integral gain;
+        accuracy (float): absolute accuracy value used in the validation tests.
+
+    **Description of Variables Being Tested**
+
+    This unit test checks the correctness of the output motor torque
+
+    - ``motorTorqueOutMsg``
+
+    where in this case the output is one dimensional, since the spinning body is one dimensional. The unit test
+    only checks the output at the first time step, for which the integral term does not contribute.
+    """
+    # each test method requires a single assert method to be called
+    [testResults, testMessage] = hingedRigidBodyPIDMotorTestFunction(show_plots, thetaR, thetaDotR, theta, thetaDot, K, P, I, accuracy)
+    assert testResults < 1, testMessage
+
+
+def hingedRigidBodyPIDMotorTestFunction(show_plots, thetaR, thetaDotR, theta, thetaDot, K, P, I, accuracy):
+
+    testFailCount = 0                        # zero unit test result counter
+    testMessages = []                        # create empty array to store test log messages
+    unitTaskName = "unitTask"                # arbitrary name (don't change)
+    unitProcessName = "TestProcess"          # arbitrary name (don't change)
+    bskLogging.setDefaultLogLevel(bskLogging.BSK_WARNING)
+
+    # Create a sim module as an empty container
+    unitTestSim = SimulationBaseClass.SimBaseClass()
+
+    # Create test thread
+    testProcessRate = macros.sec2nano(1)     # update process rate update time
+    testProc = unitTestSim.CreateNewProcess(unitProcessName)
+    testProc.addTask(unitTestSim.CreateNewTask(unitTaskName, testProcessRate))
+
+    # Construct algorithm and associated C++ container
+    motorConfig = hingedRigidBodyPIDMotor.hingedRigidBodyPIDMotorConfig()
+    motorWrap = unitTestSim.setModelDataWrap(motorConfig)
+    motorWrap.ModelTag = "hingedRigidBodyPIDMotor"  
+    motorConfig.K = K
+    motorConfig.P = P
+    motorConfig.I = I
+    unitTestSim.AddModelToTask(unitTaskName, motorWrap, motorConfig)
+
+    # Create input spinning body reference message
+    RefInMsgData = messaging.HingedRigidBodyMsgPayload()
```

Comment:
:small_red_triangle_down: variable name to be camelCase.

Reply:
Fixed it.